src/stateManagement/data/handlers/UpdateCellHandlerAction.ts

```diff
@@ -29,9 +29,13 @@ export default class UpdateCellHandlerAction extends AbstractTableAction<DataSta
             if (ddbbConfig.show_metadata_modified) {
                 modifiedRow[MetadataColumns.MODIFIED] = DateTime.now();
             }
+            const pathColumns: string[] =
+                ddbbConfig.group_folder_column
+                .split(",")
+                .filter(Boolean);
             // Update the row on disk
-            if (isMovingFile && ddbbConfig.group_folder_column === column.id) {
-
+            if ( isMovingFile && pathColumns.includes(column.id)) {
+                const subfolders = pathColumns .map((name) => modifiedRow[name] || "-") .join("/");
```

Comment:
We can include a validator function to check if some of the subfolders has an empty key with a Notice as feedback and just persist the new value instead and just call the updateRowFileProxy

Reply:
let me know when the PR is ready to merge

Reply:
to update config we need to use the config actions of "alterConfig" to change at the same time the value on disk and on memory.

If you change the value with view its just changed on disk (for configurations its enough because we reset the table after an edition)

About deleting or modifying a param we need to move and change all the notes afected (there is another issue for that on progress)

Reply:
We can prepare the rules with all this logic iterating this PR with https://github.com/RafaelGB/obsidian-db-folder/issues/183

With a new rule section per table . The pendign edge cases would be to move all the files in case of rename or delete

Reply:
Could you comment something on #183 ? I can assign it to you =)

About the errorHandler to move them as ungroup could be interesting. The alternative its just move them to the root of the folder

I really apreciate your proactive PR

Reply:
If it seems good to you, we can iterate on other tasks so as not to generate a very large PR.

All those considerations can be included in separate issues and I send you an invitation as contributor

Reply:
No problem with that =)

With your PR and the filter groups the next version could be fantastic!

Reply:
is it ready to merge?

Reply:
merged and tested. Works really well gj!

Could you try the group filters?